src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
fix common mistake in all places
![image](https://user-images.githubusercontent.com/11480043/235294933-10992311-2ed7-4bfc-be4f-5d768ec76998.png)
https://mate-academy.github.io/jv-program-common-mistakes/java-core/dependency-injection/dependency-injection